sections/api/typescript.md

```diff
@@ -178,6 +178,49 @@ This is the most complex example where we have specific properties for the styli
 the rest of the required properties by the **Header** through. That means when you use **Title** it will have
 the combined typing of both the styled requirements and the actual component requirements.
 
+### Using `attrs` method
+
+The `attrs` method accepts an Interpolation object, which can be either a string that will be merged into the rest of the component's props, or a function that receives the styled component's `props` as the first and only argument.
+
+For the string case, make sure to type it "twice", like this:
+
+```jsx
+import styled from 'styled-components';
+import Header from './Header';
+
+interface TitleProps {
+  readonly isActive: boolean;
+};
+
+const Title = styled(Header).attrs<TitleProps>({ isActive: true })<TitleProps>`
```

Comment:
not sure if generic props need to be specified explicitly, it should have been a partial object of the wrapped component/tag. I'd suggest using examples from the website or similar https://styled-components.com/docs/api#attrs